[Thu Feb 05 19:56:14 2015] [error] [client 127.0.0.1] client denied by server configuration: /home/developer/server/symfonysite/src/
[Thu Feb 05 19:56:14 2015] [error] [client 127.0.0.1] client denied by server configuration: /home/developer/server/symfonysite/app/
[Thu Feb 05 19:56:14 2015] [error] [client 127.0.0.1] File does not exist: /home/developer/server/symfonysite/favicon.ico
[Thu Feb 05 19:56:15 2015] [error] [client 127.0.0.1] File does not exist: /home/developer/server/symfonysite/favicon.ico
[Thu Feb 05 19:56:16 2015] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'UnexpectedValueException' with message 'The stream or file "/home/developer/server/symfonysite/app/logs/prod.log" could not be opened: failed to open stream: Permission denied' in /home/developer/server/symfonysite/app/cache/prod/classes.php:5623\nStack trace:\n#0 /home/developer/server/symfonysite/app/cache/prod/classes.php(5563): Monolog\\Handler\\StreamHandler->write(Array)\n#1 /home/developer/server/symfonysite/app/cache/prod/classes.php(5702): Monolog\\Handler\\AbstractProcessingHandler->handle(Array)\n#2 /home/developer/server/symfonysite/app/cache/prod/classes.php(6010): Monolog\\Handler\\FingersCrossedHandler->handle(Array)\n#3 /home/developer/server/symfonysite/app/cache/prod/classes.php(6081): Monolog\\Logger->addRecord(600, 'Uncaught Except...', Array)\n#4 /home/developer/server/symfonysite/app/cache/prod/classes.php(1922): Monolog\\Logger->log('emergency', 'Uncaught Except...', Array)\n#5 [internal function]: Symfony\\Component\\Debug\\ErrorHandler->handleException(Object(UnexpectedValueExceptio in /home/developer/server/symfonysite/app/cache/prod/classes.php on line 5623, referer: http://symfonysite/
[Thu Feb 05 19:56:16 2015] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'UnexpectedValueException' with message 'The stream or file "/home/developer/server/symfonysite/app/logs/prod.log" could not be opened: failed to open stream: Permission denied' in /home/developer/server/symfonysite/app/cache/prod/classes.php:5623\nStack trace:\n#0 /home/developer/server/symfonysite/app/cache/prod/classes.php(5563): Monolog\\Handler\\StreamHandler->write(Array)\n#1 /home/developer/server/symfonysite/app/cache/prod/classes.php(5702): Monolog\\Handler\\AbstractProcessingHandler->handle(Array)\n#2 /home/developer/server/symfonysite/app/cache/prod/classes.php(6010): Monolog\\Handler\\FingersCrossedHandler->handle(Array)\n#3 /home/developer/server/symfonysite/app/cache/prod/classes.php(6081): Monolog\\Logger->addRecord(600, 'Fatal Error: Un...', Array)\n#4 /home/developer/server/symfonysite/app/cache/prod/classes.php(1922): Monolog\\Logger->log('emergency', 'Fatal Error: Un...', Array)\n#5 /home/developer/server/symfonysite/app/cache/prod/classes.php(1969): Symfony\\Component\\Debug\\ErrorHandler in /home/developer/server/symfonysite/app/cache/prod/classes.php on line 5623, referer: http://symfonysite/
[Thu Feb 05 19:56:19 2015] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'UnexpectedValueException' with message 'The stream or file "/home/developer/server/symfonysite/app/logs/prod.log" could not be opened: failed to open stream: Permission denied' in /home/developer/server/symfonysite/app/cache/prod/classes.php:5623\nStack trace:\n#0 /home/developer/server/symfonysite/app/cache/prod/classes.php(5563): Monolog\\Handler\\StreamHandler->write(Array)\n#1 /home/developer/server/symfonysite/app/cache/prod/classes.php(5702): Monolog\\Handler\\AbstractProcessingHandler->handle(Array)\n#2 /home/developer/server/symfonysite/app/cache/prod/classes.php(6010): Monolog\\Handler\\FingersCrossedHandler->handle(Array)\n#3 /home/developer/server/symfonysite/app/cache/prod/classes.php(6081): Monolog\\Logger->addRecord(600, 'Uncaught Except...', Array)\n#4 /home/developer/server/symfonysite/app/cache/prod/classes.php(1922): Monolog\\Logger->log('emergency', 'Uncaught Except...', Array)\n#5 [internal function]: Symfony\\Component\\Debug\\ErrorHandler->handleException(Object(UnexpectedValueExceptio in /home/developer/server/symfonysite/app/cache/prod/classes.php on line 5623, referer: http://symfonysite/
[Thu Feb 05 19:56:19 2015] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'UnexpectedValueException' with message 'The stream or file "/home/developer/server/symfonysite/app/logs/prod.log" could not be opened: failed to open stream: Permission denied' in /home/developer/server/symfonysite/app/cache/prod/classes.php:5623\nStack trace:\n#0 /home/developer/server/symfonysite/app/cache/prod/classes.php(5563): Monolog\\Handler\\StreamHandler->write(Array)\n#1 /home/developer/server/symfonysite/app/cache/prod/classes.php(5702): Monolog\\Handler\\AbstractProcessingHandler->handle(Array)\n#2 /home/developer/server/symfonysite/app/cache/prod/classes.php(6010): Monolog\\Handler\\FingersCrossedHandler->handle(Array)\n#3 /home/developer/server/symfonysite/app/cache/prod/classes.php(6081): Monolog\\Logger->addRecord(600, 'Fatal Error: Un...', Array)\n#4 /home/developer/server/symfonysite/app/cache/prod/classes.php(1922): Monolog\\Logger->log('emergency', 'Fatal Error: Un...', Array)\n#5 /home/developer/server/symfonysite/app/cache/prod/classes.php(1969): Symfony\\Component\\Debug\\ErrorHandler in /home/developer/server/symfonysite/app/cache/prod/classes.php on line 5623, referer: http://symfonysite/
